src/context/team.py

```diff
@@ -0,0 +1,75 @@
+import discord
+
+from typing import List
+
+from src.queries.player import get_player, remove_player
+from src.queries.team import get_team, get_team_members, remove_team
+
+from src.models.player import Player
+
+
+async def get_team_channels(guild: discord.Guild, team_name: str):
+    team = await get_team(team_name)
+
+    voice_channel = guild.get_channel(int(team.voice_channel_id))
+    text_channel = guild.get_channel(int(team.text_channel_id))
+    category_channel = guild.get_channel(int(team.category_channel_id))
```

Comment:
don't need casting here

Reply:
whoops forgot to change that after testing